app/assets/javascripts/revised/pages/info/manufacturers.js.coffee

```diff
@@ -1,13 +1,11 @@
-class BikeIndex.Views.ContentManufacturers extends Backbone.View
-
-  initialize: ->
+class BikeIndex.InfoManufacturers extends BikeIndex
+  constructor: ->
```

Comment:
Similarly, because no backbone - `constructor` is the method that is called on initialization, unlike `initialize`, which is a fancy backbone thing.